addok/helpers/serializers.py

```diff
@@ -0,0 +1,14 @@
+import json
+import marshal
+import zlib
+
+
+class Compressed:
```

Comment:
> Why isn't it named ZLibSerializer?

My idea was to have a "generic" compressed class, that we could change without renaming it if we find a better algo at some point, but I'll follow your advice.

> What about inheriting from a BaseSerializer which implements dumps/loads

I feel like the `dumps`/`loads` API is already a sort of "convention": json (and family), yaml, msgpack, pickle, marshal… are all following this convention, so I feel there is no need from my side to enforce this.